### PR TITLE
Pull request test

### PR DESCRIPTION
**Pull request by Gopxfs**
This is my first pull request, learning how to use Linters and the GitHubFlow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
...
+# Hello-Microverse
+> This is a project to test the basics of creating a new repository.
+
+## Built WIth
+- Html
+- Css
+
+## Authors:
+:bust_in_silhouette:Gabriel Fonseca
+- Github: @Gopxfs
```
Comment (ShahierNashaat) [inline README.md]: - [x] Please make your readme file like the Microverse README template to make it more descriptive.
- Add a screenshot of your webpage.
- Getting Start section like if someone cloned your repo how should he run it.
- In the author section add a link to your Github profile and a link to your Linkedin profile.
- Show your support section.
- Acknowledgments section.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+test.md
+.hintrc
+.stylelintrc.json
```
Comment (ShahierNashaat) [inline .gitignore]: - [x] Please add the node_modules to your .gitignore file.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Gopxfs , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Please write a description of your pull request.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Gopxfs) [reply README.md]: Hey @ShahierNashaat ! Did everything you told! Thanks.

File: .gitignore (hunk 2 above)
Comment (Gopxfs) [reply .gitignore]: Hey @ShahierNashaat ! Updated the .gitignore file. Thank you.

**Comment by Gopxfs** [comment]
This is my second pull request! Hope everything is right.

**Review by youmari — COMMENTED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Gopxfs 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.

- [ ] [OPTIONAL] I see that you are you using the past when writing your commits please next time try to follow these [rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Cheers and Happy coding!👏👏👏
-----

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification. 
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**